crates/runner/src/check.rs

```
@@ -10,15 +10,15 @@ use tracing::{info, instrument};
 
 use crate::make_env;
 
-pub(crate) static REGISTRY_PATCHES: LazyLock<PathBuf> =
+pub static REGISTRY_PATCHES: LazyLock<PathBuf> =
     LazyLock::new(|| PathBuf::from(env!("REGISTRY_PATCHES")));
```

Clippy (1.97.1)

[error] environment variable `REGISTRY_PATCHES` not defined at compile time

(error)